interface/YamiVersion.h.in

```diff
@@ -22,6 +22,15 @@
 #ifdef __cplusplus
 extern "C" {
 #endif
+/**
+ * __BUILD_DECODER__: represent wheter decoder subdirectory is compiled or not.
+ * __BUILD_ENCODER__: represent wheter encoder subdirectory is compiled or not.
+ * __BUILD_VPP__: represent wheter vpp subdirectory is compiled or not.
+ *
+ */
+#define __BUILD_DECODER__    @__BUILD_DECODER__@
```

Comment:
Perhaps something like `#define __YAMI_HAS_DECODER__ @__BUILD_DECODER__@` (etc.) would be less generic and reduce the chance of name clashes in client projects.  Also, I think these should go in Yami.h.in, instead, and then generate Yami.h since they aren't really related to versioning.